lib/ansible/modules/windows/win_chocolatey.ps1

```diff
@@ -207,6 +211,16 @@ Function Choco-Upgrade
         $options += "--source", $source
     }
 
+    if ($username)
```

Comment:
We need to ensure these parameters are only provided when the `source` parameter was set. It should report if this is not the case.

Reply:
(Same below)